components/ryr-instructions.js

```diff
@@ -0,0 +1,120 @@
+import PropTypes from 'prop-types';
+
+const RyrInstructions = () => {
+  return (
+    <div>
+      <div id="instructions" className="color-black">
+        <h2 className="center">What to do next?</h2>
+        <div className="instruction-list">
+          <ol className="list-numbered">
+            <li>
+              Send your email to{' '}
+              <a href="mailto:bicycleprogram@austintexas.gov?subject=Bicycle Corral Request">
+                bicycleprogram@austintexas.gov
+              </a>
+            </li>
+            <li>Set &quot;Bicycle Corral Request&quot; as the subject.</li>
+          </ol>
+        </div>
+      </div>
+      <style jsx>
+        {`
+      h2 {
+        text-align: center;
+        margin-bottom: 1em;
+      }
+      
+      section {
+        padding: 15px;
+      }
+
+      #instructions {
```

Comment:
There is no #instructions in this component.

Reply:
Fixed and removed section

Reply:
Removed this section

Reply:
Removed this section